pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -738,25 +719,24 @@ func TestUpdatePodAndGet(t *testing.T) {
 		makeBasePod(t, nodeName, "test", "200m", "1Ki", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 8080, Protocol: "TCP"}}),
 	}
 	tests := []struct {
-		pod *v1.Pod
-
+		name        string
+		pod         *v1.Pod
 		podToUpdate *v1.Pod
 		handler     func(logger klog.Logger, cache Cache, pod *v1.Pod) error
-
-		assumePod bool
+		assumePod   bool
 	}{
 		{
-			pod: testPods[0],
-
+			name:        "do not update pod",
```

Comment:
Could you add more detail? like "do not update pod when the pod is hogehoge". Same in the below test case

Reply:
OK, I will update them with more details.

Reply:
Done.